### DISCUSSION TITLE
How to deal with very large datasets?

### DISCUSSION THREAD

**Discussion by davidgasquez** [datonic/hub#29] [Q&A]
Talking about 1TB datasets. They'll need to be [partitioned](https://github.com/orgs/datonic/discussions/31).

**Comment by TomNicholas** [comment]
Go down this rabbit hole and you end up with chunked [cloud-native big data repositories](https://medium.com/pangeo/step-by-step-guide-to-building-a-big-data-portal-e262af1c2977). The latest and greatest technology for this is [Icechunk](https://icechunk.io/en/latest/) and [Iceberg](https://iceberg.apache.org/).

**Reply by davidgasquez** [reply]
I've been there! :sweat_smile: Didn't document much but ended up building some cool stuff on top of Delta/Iceberg, IPFS, and DuckDB. 

This discussion is something I haven't thought a lot since there are already great solutions as you mention. 

I need to do another deep dive into catalogs since lots of things have happened in the last few months!
